src/libraries/apicompat/ApiCompatBaseline.NetCoreAppLatestStable.xml

```diff
@@ -2551,6 +2551,12 @@
     <Left>net7.0/System.Linq.Queryable.dll</Left>
     <Right>net8.0/System.Linq.Queryable.dll</Right>
   </Suppression>
+  <Suppression>
```

Comment:
What's this change for?

Reply:
I moved the RDC annotation from the type level to the individual constructors, so an ApiCompat suppression is necessary. This lets me reference the type from JsonMetadataServices without warnings.